src/utility/getProductsById.js

```diff
@@ -0,0 +1,6 @@
+export default function getProductsById(id, products) {
+  const foundProduct = products.filter((product) => {
```

Comment:
You can just use the `.find` method instead of filtering
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find